### PR TITLE
Gui testcases 3075

### PR DESCRIPTION
**Pull request by maryvictol**
This PR adds testcases for Audit pipe mount operations #3075 and Data access audit #3059

### CODE REVIEW

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_1.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+| 4 | Launch tool with *Default settings* | |
+| 5 | At the **Runs** page, click the just-launched run | |
+| 6 | Wait until the **SSH** hyperlink appears | |
+| 7 | Click the **SSH** hyperlink | |
+| 8 | In the opened tab, enter and perform the `pipe storage cp storage1/file1 storage2/` command |  |
+| 9 | Logout and login as the admin user from the prerequisites | |
+| 10 | Open the **Settings** page | |
+| 11 | Open the **SYSTEM MANAGEMENT** tab | |
+| 12 | Select the **LOGS** tab | |
+| 13 | Click ***Show advanced*** button | |
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_1.md:32]: I suggest to add an action like "Clear the field **From**" - as by default logs are shown from the moment of the opening **Logs** form.

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_2.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+| 1 | Login as the non-admin user from the prerequisites | |
+| 2 | At the **Runs** page, click the run launched at step 4 of [3059_1](3059_1.md) case | |
+| 3 | Wait until the **SSH** hyperlink appears | |
+| 4 | Click the **SSH** hyperlink | |
+| 5 | In the opened tab, enter and perform the `pipe storage cp -r storage1/folder1 storage2/folder1` command | |
+| 6 | Logout and login as the admin user from the prerequisites | |
+| 7 | Open the **Settings** page | |
+| 8 | Open the **SYSTEM MANAGEMENT** page | |
+| 9 | Select the **LOGS** tab | |
+| 10 | Click ***Show advanced*** button | |
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_2.md:21]: I suggest to add an action like "Clear the field **From**" - as by default logs are shown from the moment of the opening **Logs** form.

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_3.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+| 30 | Click **Download** button near the `file1` name | |
+| 31 | Logout and _Check audit log_ (see below) | Record <br> `READ storage3/file1` <br> is shown in the top of logs records |
+
+\* _Check audit log_:
+1. Login as the admin user from the prerequisites
+2. Open the **Settings** page
+3. Open the **SYSTEM MANAGEMENT** page
+4. Select the **LOGS** tab
+5. Click ***Show advanced*** button
+6. Click **Type** field and select `audit` value from drop-down
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_3.md:55]: I suggest to add an action like "Clear the field **From**" - as by default logs are shown from the moment of the opening **Logs** form.

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+| 4 | Launch tool with *Default settings* | |
+| 5 | At the **Runs** page, click the just-launched run | |
+| 6 | Wait until the **SSH** hyperlink appears | |
+| 7 | Click the **SSH** hyperlink | |
+| 8 | In the opened tab, enter and perform the `pipe storage mount -f -m 775 -o allow_other <destination_folder>` command | |
+| 9 | Enter and perform the `echo "test info" >> <destination_folder>/storage1/file3` command | |
+| 10 | Logout and _Check audit log_ (see below) | Record <br> `WRITE cp://storage1/file3` <br> is shown in the top of logs records |
+| 11 | Repeat steps 5-9 for `cp <destination_folder>/storage1/file1 <destination_folder>/storage2/file1` command | Records <br>`READ cp://storage1/file1` <br> `WRITE cp://storage2/file1` <br> are shown in the top of logs records |
+| 12 |Repeat steps 5-9 for `cat <destination_folder>/storage1/file2` command | Records <br>`READ cp://storage1/file1` <br> is shown in the top of logs records |
+| 13 | Repeat steps 5-9 for `mv <destination_folder>/storage1/file2 <destination_folder>/storage2` command | Record <br> `MOVE cp://storage1/file2 cp://storage2/file2` <br> is shown in the top of logs records |
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md:37]: Is it correct record `MOVE` in the **Logs**?

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+| 3 | Select any tool | |
+| 4 | Launch tool with *Default settings* | |
+| 5 | At the **Runs** page, click the just-launched run | |
+| 6 | Wait until the **SSH** hyperlink appears | |
+| 7 | Click the **SSH** hyperlink | |
+| 8 | In the opened tab, enter and perform the `pipe storage mount -f -m 775 -o allow_other <destination_folder>` command | |
+| 9 | Enter and perform the `echo "test info" >> <destination_folder>/storage1/file3` command | |
+| 10 | Logout and _Check audit log_ (see below) | Record <br> `WRITE cp://storage1/file3` <br> is shown in the top of logs records |
+| 11 | Repeat steps 5-9 for `cp <destination_folder>/storage1/file1 <destination_folder>/storage2/file1` command | Records <br>`READ cp://storage1/file1` <br> `WRITE cp://storage2/file1` <br> are shown in the top of logs records |
+| 12 |Repeat steps 5-9 for `cat <destination_folder>/storage1/file2` command | Records <br>`READ cp://storage1/file1` <br> is shown in the top of logs records |
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md]: I suppose, record `READ cp://storage1/file2` shall be in **Logs**.

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+| 13 | Repeat steps 5-9 for `mv <destination_folder>/storage1/file2 <destination_folder>/storage2` command | Record <br> `MOVE cp://storage1/file2 cp://storage2/file2` <br> is shown in the top of logs records |
+| 14 | Repeat steps 5-9 for `rm <destination_folder>/storage2/file2` command | Record <br> `DELETE cp://storage2/file2` <br> is shown in the top of logs records |
+| 15 | Repeat steps 5-9 for `cp -rf <destination_folder>/storage1/folder1 <destination_folder>/storage2/folder1` command | For each files from folder1 (*inner_file1* and *inner_file2*) records <br>`READ cp://storage1/folder1/<file>` <br> `WRITE cp://storage2/folder1/<file>` <br> are shown in the top of logs records |
+| 16 | Repeat steps 5-9 for `mv -f <destination_folder>/storage1/folder2 <destination_folder>/storage2/folder2` command | For each files from folder2 (*inner_file3* and *inner_file4*) records <br> `READ cp://storage1/folder2/<file>` <br> `WRITE cp://storage2/folder2/<file>` <br> `DELETE cp://storage2/folder2/<file>` <br> are shown in the top of logs records |
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md]: I suppose, record `DELETE cp://storage1/folder2/<file>` shall be in **Logs**.

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+| 17 | Repeat steps 5-9 for `rm -f  <destination_folder>/storage1/folder1` command | For each files from folder1 (*inner_file1* and *inner_file2*) record <br> `DELETE cp://storage1/folder2/<file>` <br> are shown in the top of logs records |
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md]: I suppose, record `DELETE cp://storage1/folder1/<file>` shall be in **Logs**.

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+
+\* _Check audit log_:
+1. Login as the admin user from the prerequisites | |
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md]: Steps 1-5 - remove extra symbols (`|  |`)

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+2. Open the **Settings** page | |
+3. Open the **SYSTEM MANAGEMENT** page | |
+4. Select the **LOGS** tab | |
+5. Click ***Show advanced*** button | |
+6. Click **Type** field and select `audit` value from drop-down
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md:50]: I suggest to add an action like "Clear the field **From**" - as by default logs are shown from the moment of the opening **Logs** form.

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_3.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+| 23 | Repeat steps 1-2 | |
+| 24 | Click **Remove** button near the `file2` name and confirm removing | |
+| 25 | Logout and _Check audit log_ (see below) | Record <br> `DELETE storage3/file2` <br> is shown in the top of logs records |
+| 26 | Repeat steps 1-2 | |
+| 27 | Click **Remove** button near the `folder1` name and confirm removing | |
+| 28 | Logout and _Check audit log_ (see below) | For each files from `folder1` (*inner_file1* and *inner_file2*) records <br> `DELETE storage3/folder1/<file>` <br> are shown in the top of logs records |
+| 29 | Repeat steps 1-2 | |
+| 30 | Click **Download** button near the `file1` name | |
+| 31 | Logout and _Check audit log_ (see below) | Record <br> `READ storage3/file1` <br> is shown in the top of logs records |
+
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_3.md:48]: Add the section **After** in which the storage created at the Preparations shall be removed.

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+| 14 | Repeat steps 5-9 for `rm <destination_folder>/storage2/file2` command | Record <br> `DELETE cp://storage2/file2` <br> is shown in the top of logs records |
+| 15 | Repeat steps 5-9 for `cp -rf <destination_folder>/storage1/folder1 <destination_folder>/storage2/folder1` command | For each files from folder1 (*inner_file1* and *inner_file2*) records <br>`READ cp://storage1/folder1/<file>` <br> `WRITE cp://storage2/folder1/<file>` <br> are shown in the top of logs records |
+| 16 | Repeat steps 5-9 for `mv -f <destination_folder>/storage1/folder2 <destination_folder>/storage2/folder2` command | For each files from folder2 (*inner_file3* and *inner_file4*) records <br> `READ cp://storage1/folder2/<file>` <br> `WRITE cp://storage2/folder2/<file>` <br> `DELETE cp://storage2/folder2/<file>` <br> are shown in the top of logs records |
+| 17 | Repeat steps 5-9 for `rm -f  <destination_folder>/storage1/folder1` command | For each files from folder1 (*inner_file1* and *inner_file2*) record <br> `DELETE cp://storage1/folder2/<file>` <br> are shown in the top of logs records |
+
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md:42]: Add the section **After** in which the storages created at the Preparations shall be removed.

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_5.md [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+| 16 | Repeat step 1 | |
+| 17 | Click **Remove** button near the `file2` name and confirm removing | |
+| 18 | Logout and _Check audit log_ (see below) | Record <br> `DELETE storage1/file2` <br> is shown in the top of logs records |
+| 19 | Repeat steps 1-2 | |
+| 20 | Click **Remove** button near the `folder1` name and confirm removing | |
+| 21 | Logout and _Check audit log_ (see below) | For each files from `folder1` (*inner_file1* and *inner_file2*) records <br> `DELETE storage1/folder1_new/<file>` <br> are shown in the top of logs records |
+
+
+\* _Check audit log_:
+1. Login as the admin user from the prerequisites | |
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_5.md]: Steps 1-5 - remove extra symbols (`|  |`)

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_5.md [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+2. Open the **Settings** page | |
+3. Open the **SYSTEM MANAGEMENT** page | |
+4. Select the **LOGS** tab | |
+5. Click ***Show advanced*** button | |
+6. Click **Type** field and select `audit` value from drop-down
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_5.md:50]: I suggest to add an action like "Clear the field **From**" - as by default logs are shown from the moment of the opening **Logs** form.

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_5.md [hunk 14]
```diff
@@ -0,0 +1,52 @@
...
+| 13 | Click **Rename** button opposite `folder1` | |
+| 14 | Specify a new value in the _Name_ field. Click **OK** button | |
+| 15 | Logout and _Check audit log_ (see below) | For each files from `folder1` (*inner_file1* and *inner_file2*) records <br> `READ storage1/folder1/<file>` <br> `WRITE storage1/folder1_new/<file>` <br> `DELETE storage1/folder1/<file>` <br> are shown in the top of logs records | 
+| 16 | Repeat step 1 | |
+| 17 | Click **Remove** button near the `file2` name and confirm removing | |
+| 18 | Logout and _Check audit log_ (see below) | Record <br> `DELETE storage1/file2` <br> is shown in the top of logs records |
+| 19 | Repeat steps 1-2 | |
+| 20 | Click **Remove** button near the `folder1` name and confirm removing | |
+| 21 | Logout and _Check audit log_ (see below) | For each files from `folder1` (*inner_file1* and *inner_file2*) records <br> `DELETE storage1/folder1_new/<file>` <br> are shown in the top of logs records |
+
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_5.md:42]: Add the section **After** in which the storage created at the Preparations shall be removed.

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_6.md [hunk 15]
```diff
@@ -0,0 +1,31 @@
...
+3. In the `storage1`
+    - Add `file1` and `file2` to the storage root
+    - Create `folder1` with 2 files (`inner_file1` and `inner_file2`)
+    - Create `folder2` with 2 files (`inner_file3` and `inner_file4`)
+
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Open the **Library** page | |
+| 2 | Open the object storage `storage3` created at step 2 of the Preparations | |
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_6.md]: Fox the name of the object storage to `storage1`

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_6.md [hunk 16]
```diff
@@ -0,0 +1,31 @@
...
+| 3 | Mark checkboxes near `file1` and `folder1` | |
+| 4 | Click ***Share 2 items*** button | Pop up ***Select users / groups to share 2 items*** appears |
+| 5 | Add non-admin user to list of selected user | |
+| 6 | Give **Read** permissions | |
+| 7 | Click **SAVE AND SHARE** button | Pop up with Link to 2 shared items appears |
+| 8 | Copy share link to clipboard | |
+| 9 | Logout | |
+| 10 | Open link stored at the step 5 of the Preparations. Login as the non-admin user from the prerequisites | The page with the header ***storage1-share-1*** appears that contains only `file1` and `folder1` created at step 3 of the Preparations |
+| 11 | Click **Download** button near the `file1` name | |
+| 12 | Logout and _Check audit log_ (see below) | Record <br> `READ storage1/file1` <br> is shown in the top of logs records |
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_6.md:31]: Add the section **After** in which the storage created at the Preparations shall be removed.

Comment (NShaforostov) [inline docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_6.md:31]: Add _Check audit log_ section - similar to the one in tests 3059_3-3059_5

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_2.md [hunk 17]
```diff
@@ -0,0 +1,28 @@
...
+| 11 | Click **Type** field and select `audit` value from drop-down | For each files from folder1 (*inner_file1* and *inner_file2*) records <br>`READ storage1/folder1/<file>` <br> `WRITE storage2/folder1/<file>`  <br> are shown in the top of logs records |
+| 12 | Logout | |
+| 13 | Repeat steps 1-12 for `pipe storage -r mv storage1/folder2 storage2/folder2` command | For each files from folder2 (*inner_file3* and *inner_file4*) records <br> `READ storage1/folder2/<file>` <br> `WRITE storage2/folder2/<file>` <br> `DELETE storage1/folder2/<file>` <br> are shown in the top of logs records |
+| 14 | Repeat steps 1-12 for `pipe storage rm -r storage1/folder1 -y` command | For each files from folder1 (*inner_file1* and *inner_file2*) record <br> `DELETE storage1/folder1/<file>` <br> is shown in the top of logs records |
+
+**After:**
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_2.md]: I suggest to remove the section **After**, as this run is still need in the case 3075

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3075.md [hunk 18]
```diff
@@ -0,0 +1,29 @@
...
+| 2 | At the Runs page, click the run launched at step 4 of [3059_1](3059_1.md) case | |
+| 3 | Wait until the SSH hyperlink appears | |
+| 4 | Click the SSH hyperlink | |
+| 5 | In the opened tab, enter and perform the `echo "test info" >> cloud-data/storage2/file3` command | |
+| 6 | Logout and login as the admin user from the prerequisites | |
+| 7 | Open the **Settings** page | |
+| 8 | Open the **SYSTEM MANAGEMENT** page | |
+| 9 | Select the **LOGS** tab | |
+| 10 | Click ***Show advanced*** button | |
+| 11 | Click **Type** field and select `audit` value from drop-down | Record <br> `WRITE storage2/file3` <br> is shown in the top of logs records |
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3075.md:22]: I suggest to add an action like "Clear the field **From**" - as by default logs are shown from the moment of the opening **Logs** form.

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3075.md [hunk 19]
```diff
@@ -0,0 +1,29 @@
...
+| 12 | Logout | |
+| 13 | Repeat steps 1-12 for `cp -rf cloud-data/storage2/folder1 cloud-data/storage1/folder1` command | For each files from folder1 (*inner_file1* and *inner_file2*) records <br>`READ storage2/folder1/<file>` <br> `WRITE storage1/folder1/<file>` <br> are shown in the top of logs records |
+| 14 | Repeat steps 1-12 for `mv -f cloud-data/storage2/folder2 cloud-data/storage1/folder2` command | For each files from folder2 (*inner_file3* and *inner_file4*) records <br> `READ storage2/folder2/<file>` <br> `WRITE storage1/folder2/<file>` <br> `DELETE storage2/folder2/<file>` <br> are shown in the top of logs records |
+| 15 | Repeat steps 1-12 for `rm -f cloud-data/storage2/file2` command | Record <br> `DELETE storage2/file2` <br> is shown in the top of logs records |
+
+**After:**
+- Stop run launched at step 4 of [3059_2](3059_2.md) case
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3075.md]: Add the item:
"- Delete storages created at step 2 in the Preparations of [3059_1](3059_1.md) case"

**Review by NShaforostov — APPROVED** [review]
Please, fix minor notes before the merge (see details in comments).

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md (hunk 8 above)
Comment (maryvictol) [reply docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md]: Done

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3075.md (hunk 19 above)
Comment (maryvictol) [reply docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3075.md]: Done

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_2.md (hunk 17 above)
Comment (maryvictol) [reply docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_2.md]: Done

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_6.md (hunk 16 above)
Comment (maryvictol) [reply docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_6.md:31]: Done

Comment (maryvictol) [reply docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_6.md:31]: Done

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_6.md (hunk 15 above)
Comment (maryvictol) [reply docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_6.md]: Done

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_5.md (hunk 12 above)
Comment (maryvictol) [reply docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_5.md]: Done

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_5.md (hunk 14 above)
Comment (maryvictol) [reply docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_5.md:42]: Done

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md (hunk 11 above)
Comment (maryvictol) [reply docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md:42]: Done

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_3.md (hunk 10 above)
Comment (maryvictol) [reply docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_3.md:48]: Done

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md (hunk 5 above)
Comment (maryvictol) [reply docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md]: Done

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md (hunk 6 above)
Comment (maryvictol) [reply docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md]: Done

File: docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md (hunk 7 above)
Comment (maryvictol) [reply docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_4.md]: Done

**Pull request merged**